apps/vr-tests-web-components/src/stories/tabs/tabs-interactive.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28931&commitId=14d783eb0cb5b45124115671a465368742088c55&env=prod&clientType=FLUENTUI)


  <details><summary><b>Tabs</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tabs.Appearance Subtle.hover.chromium.png|0| Added|
|Tabs.Appearance Subtle.normal.chromium.png|0| Added|
|Tabs.Appearance Transparent.hover.chromium.png|0| Added|
|Tabs.Appearance Transparent.normal.chromium.png|0| Added|
|Tabs.Default - Dark Mode.normal.chromium.png|0| Added|
|Tabs.Default - RTL.normal.chromium.png|0| Added|
|Tabs.Default.normal.chromium.png|0| Added|
|Tabs.Disabled - Dark Mode.normal.chromium.png|0| Added|
|Tabs.Disabled.normal.chromium.png|0| Added|
|Tabs.Horizontal.normal.chromium.png|0| Added|
|Tabs.Navigation - RTL.2nd tab click.chromium.png|0| Added|
|Tabs.Navigation - RTL.hover.chromium.png|0| Added|
|Tabs.Navigation - RTL.normal.chromium.png|0| Added|
|Tabs.Navigation - RTL.right arrow.chromium.png|0| Added|
|Tabs.Navigation Over Disabled Item.2nd tab click.chromium.png|0| Added|
|Tabs.Navigation Over Disabled Item.hover.chromium.png|0| Added|
|Tabs.Navigation Over Disabled Item.normal.chromium.png|0| Added|
|Tabs.Navigation Over Disabled Item.right arrow.chromium.png|0| Added|
|Tabs.Navigation Vertical.chromium.png|0| Added|
|Tabs.Navigation.2nd tab click.chromium.png|0| Added|
|Tabs.Navigation.down arrow.chromium.png|0| Added|
|Tabs.Navigation.hover.chromium.png|0| Added|
|Tabs.Navigation.normal.chromium.png|0| Added|
|Tabs.Size Large - Dark Mode.normal.chromium.png|0| Added|
|Tabs.Size Large Vertical - Dark Mode.normal.chromium.png|0| Added|
|Tabs.Size Large Vertical.normal.chromium.png|0| Added|
|Tabs.Size Large.normal.chromium.png|0| Added|
|Tabs.Size Medium - Dark Mode.normal.chromium.png|0| Added|
|Tabs.Size Medium Vertical - Dark Mode.normal.chromium.png|0| Added|
|Tabs.Size Medium Vertical.normal.chromium.png|0| Added|
|Tabs.Size Medium.normal.chromium.png|0| Added|
|Tabs.Size Small - Dark Mode.normal.chromium.png|0| Added|
|Tabs.Size Small Vertical - Dark Mode.normal.chromium.png|0| Added|
|Tabs.Size Small Vertical.normal.chromium.png|0| Added|
|Tabs.Size Small.normal.chromium.png|0| Added|
|Tabs.Vertical.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>